NuGet.Config

```diff
@@ -5,8 +5,9 @@
     <clear />
     <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="AspNetCIDev" value="https://www.myget.org/F/aspnetvnext/api/v3/index.json" />
```

Comment:
Not going to merge this in, but I'm not sure how to get these packages on to cli-deps